bin/run-ansible.sh

```diff
@@ -366,15 +366,20 @@ fetch_ansible_files
 install_pip_packages
 
 
-# Fix for WSL - THIS IS NOT SUPPORTED AT ALL!!!
+# Fix for WSL - THIS IS NOT SUPPORTED AT ALL - Do not read this, don't look at it, don't try to run it, we don't support it
 if grep -q "microsoft" /proc/version  && grep -iq "wsl" /proc/version; then
   # python3 -m pip install pip --upgrade
   pip install pyopenssl --upgrade
+  WSL_START_DOCKER_COMMAND='wsl.exe --distribution "${WSL_DISTRO_NAME}" --user root --exec /usr/sbin/service docker start'
   if [[ $(which docker | wc -l) -gt 0 ]]; then
     if service docker status 2>&1 | grep -q "is not running"; then
+      ${WSL_START_DOCKER_COMMAND}
       wsl.exe --distribution "${WSL_DISTRO_NAME}" --user root --exec /usr/sbin/service docker start
     fi
   fi
+  if [[ $(cat ~/.bashrc  | grep "wsl.exe --distribution \"\${WSL_DISTRO_NAME}\" --user root --exec /usr/sbin/service docker start" | wc -l) -eq 0 ]]; then
+    echo "wsl.exe --distribution \"\${WSL_DISTRO_NAME}\" --user root --exec /usr/sbin/service docker start" >> ~/.bashrc
```

Comment:
could you do 
```suggestion
    echo  ${WSL_START_DOCKER_COMMAND} >> ~/.bashrc
```
?

Reply:
Couldn't make that work, not sure why :D

Reply:
That should work, if you define `WSL_START_DOCKER_COMMAND` as `"wsl.exe --distribution \"${WSL_DISTRO_NAME}\" --user root --exec /usr/sbin/service docker start"`.

Using single quotations prevents the string from being expanded.

Reply:
Should be able to do the same in the if statement in line 380 right above.